### ISSUE TITLE
Weight Normalization: A Simple Reparameterization to Accelerate Training of Deep Neural Networks

### ISSUE THREAD

**Issue by t-oura** [arXivTimes/arXivTimes#135]
## 一言でいうと
DNNの重み更新時に重みベクトルをノルムと正規化したベクトルの向きにパラメトライズしそれぞれの勾配から学習するweight normalization(WN)を提案。識別、生成モデル、強化学習タスクでBNなどと比較評価しタスクによらず学習が高速に収束することを示した。
### 論文リンク
https://papers.nips.cc/paper/6114-weight-normalization-a-simple-reparameterization-to-accelerate-training-of-deep-neural-networks.pdf

### 著者/所属機関
Tim Salimans and Diederik P. Kingma
OpenAI
## 概要
比較対象としてbatch normalization(BN)をあげておりWNがBNの計算量の面で軽い近似になっていること説明している。計算が軽いことに加え強化学習などサンプルのノイズにセンシティブな期待される。

**Comment by t-oura** [comment]
著者によるtheano実装 https://github.com/TimSalimans/weight_norm

**Comment by t-oura** [comment]
chainerによる convolution, decovolution, full-connect層の実装
 https://github.com/musyoku/weight-normalization

**Comment by icoxfog417** [comment]
Hinton先生の講義でLayer Normalizationという似た感じのコンセプトの話がありました。Batch Normalizationはバッチサイズの影響が大きいのとRNNに効くのか明確でない、というのが背景になってます。

Layer Normalization
https://arxiv.org/pdf/1607.06450.pdf

結果としては、RNNで狙った通りBNよりも良い結果となっています(RNN以外は書かれてないので不明ですが)。

**Comment by t-oura** [comment]
Layer Normalization(LN)論文でもいちおう最後のほうにMNISTを評価しBNよりよいという結果をのせていますね。
LN、WN両論文では相互にリファーしており、LN論文ではWNとLNは似たようなものだと見做し、WN論文ではLNはRNNに適用可能なBNライクな手法の1つとして文脈で紹介しています(WNはタスクによらないという主張なので)。LNとWNを直接比較した結果はないです。
